py3status/modules/sql.py

```diff
@@ -18,15 +18,13 @@
     Parameters can be placeholders too, eg {host}, {passd}
 
 Format_row placeholders:
-    {xxx} case-sensitive placeholder name often found in the database
-    {xxx} case-sensitive placeholder name often found in the database
-    {xxx} case-sensitive placeholder name often found in the database
+    {field} name of placeholder and its value returned by the query
```

Comment:
this is still pretty lame and is not even clear to me and I know what this is about, as i said befor it is most certainly not a `name of placeholder `

Reply:
Pushedxxx.